combat-trainer.lic

```diff
@@ -2215,6 +2215,11 @@ class TrainerProcess
     when 'App Bundle'
       retreat
       bput("app my bundle quick", 'You scan', 'Appraise what', 'You cannot appraise', 'You can\'t appraise')
+    when /Summon (?<element>Aether|Air|Earth|Electricity|Fire|Water)/
```

Comment:
Should the regex include 'domain'?

Reply:
Yes, even though the game parser doesn't require the word "domain", it may be good to go ahead and have that in here in case in the future there are other "summon <foo> <bar>" that could be supported.